app/src/main/java/org/p2p/wallet/send/ui/dialogs/FreeTransactionsDetailsBottomSheet.kt

```diff
@@ -0,0 +1,45 @@
+package org.p2p.wallet.send.ui.dialogs
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.FragmentManager
+import org.p2p.uikit.utils.getColor
+import org.p2p.uikit.utils.getColorStateList
+import org.p2p.wallet.R
+import org.p2p.wallet.common.ui.bottomsheet.BaseDoneBottomSheet
+import org.p2p.wallet.databinding.DialogSendFreeTransactionsInfoBinding
+
+class FreeTransactionsDetailsBottomSheet : BaseDoneBottomSheet() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+        ) = FreeTransactionsDetailsBottomSheet().show(
```

Comment:
return type is Unit =)
since Show method: 

```
public void show(@NonNull FragmentManager manager, @Nullable String tag) {
        mDismissed = false;
        mShownByMe = true;
        FragmentTransaction ft = manager.beginTransaction();
        ft.add(this, tag);
        ft.commit();
    }
```

Reply:
then block body :)

Reply:
fixed